build.gradle

```diff
@@ -19,6 +19,7 @@ plugins {
   id 'org.sonarqube' version '3.0'
   id 'info.solidsoft.pitest' version '1.5.2'
   id 'io.freefair.lombok' version '5.1.0'
+  id 'au.com.dius.pact' version '4.0.4'
```

Comment:
Please reference the following ia-case-api branch "ConsumerContractTests". It's the most up to date PACT change for case-api.

version should be -> id 'au.com.dius.pact' version '4.1.0'